### PR TITLE
feat: add new shadowsocks-v2ray-tls servers

### PR DESCRIPTION
**Pull request by masoudkazemi**
Adding 2 new servers.

### CODE REVIEW

**Comment by Pr0nE** [comment]
Thanks,
I've tested this app but didn't work for me(VPN connects, but can't do anything)
It has Shadowsocks config but doesn't support plugins to add v2ray, how did you manage to add plugin?

**Comment by masoudkazemi** [comment]
Sorry, my mistake, as you mentioned it doesn't support the plugin. Never mind.

File: guides/shadowsocks-v2ray-tls/CONFIGS.md [hunk 1]
```diff
@@ -9,3 +9,12 @@
 # gimpmie.hair
 ## Android:
 ss://YWVzLTI1Ni1nY206TWFoc2FBbWluaQ@gimpmie.hair:443?plugin=v2ray-plugin%3Bpath%3Drabbitmq%3Bhost%3Dgimpmie.hair%3Btls
+
+
+# tmug.ir
+## Android and iOS:
+ss://YWVzLTI1Ni1nY206OTVDRTdCNzQtQ0Y4Ni00NUQyLUE0NEUtQzAzOUE1QkQxMjY4@tmug.ir:443?plugin=v2ray-plugin%3Bpath%3D%2Frabbitmq%3Bloglevel%3Dnone%3Bhost%3Dtmug.ir%3Btls
```
Comment (0xland) [inline guides/shadowsocks-v2ray-tls/CONFIGS.md]: Unfortunately These links will not work on IOS. 
IOS formats are different:
this is your IOS link:
`ss://YWVzLTI1Ni1nY206OTVDRTdCNzQtQ0Y4Ni00NUQyLUE0NEUtQzAzOUE1QkQxMjY4QHRtdWcuaXI6NDQz?tfo=1&v2ray-plugin=eyJwYXRoIjoiXC9yYWJiaXRtcSIsIm11eCI6dHJ1ZSwidGZvIjp0cnVlLCJob3N0IjoidG11Zy5pciIsIm1vZGUiOiJ3ZWJzb2NrZXQiLCJ0bHMiOnRydWV9`

File: guides/shadowsocks-v2ray-tls/CONFIGS.md [hunk 2]
```diff
@@ -9,3 +9,12 @@
...
+
+# golechai.ir
+## Android and iOS:
+ss://YWVzLTI1Ni1nY206OTVDRTdCNzQtQ0Y4Ni00NUQyLUE0NEUtQzAzOUE1QkQxMjY4@golechai.ir:443?plugin=v2ray-plugin%3Bpath%3D%2Frabbitmq%3Bloglevel%3Dnone%3Bhost%3Dgolechai.ir%3Btls
```
Comment (0xland) [inline guides/shadowsocks-v2ray-tls/CONFIGS.md]: `ss://YWVzLTI1Ni1nY206OTVDRTdCNzQtQ0Y4Ni00NUQyLUE0NEUtQzAzOUE1QkQxMjY4QGdvbGVjaGFpLmlyOjQ0Mw?tfo=1&v2ray-plugin=eyJwYXRoIjoiXC9yYWJiaXRtcSIsIm11eCI6dHJ1ZSwidGZvIjp0cnVlLCJob3N0IjoiZ29sZWNoYWkuaXIiLCJtb2RlIjoid2Vic29ja2V0IiwidGxzIjp0cnVlfQ`

**Review by 0xland — CHANGES_REQUESTED** [review]

File: guides/shadowsocks-v2ray-tls/CONFIGS.md (hunk 1 above)
Comment (masoudkazemi) [reply guides/shadowsocks-v2ray-tls/CONFIGS.md]: Added, thanks.

**Pull request merged**